### ISSUE TITLE
Would you consider to migrate from puppeter/node to ferrum/ruby

### ISSUE THREAD

**Issue by pftg** [Studiosity/grover#162]
There is node dependency here, but this dependency could be avoided by using https://github.com/rubycdp/ferrum.

This will flex dependencies, and simplify installation and testing.

**Comment by abrom** [comment]
Sorry for the late reply @pftg, I must have missed the notification.

Note I have researched this in the past but found the Ruby CDP options out there were significantly slower than using Puppeteer (I tried a couple). It was a while ago, but I recall it being in the order of 30-40% slower.

Also, whilst I am by no means suggesting that those packages aren't well maintained, the Puppeteer library is maintained by Google so is likely to contain all the latest features and be well supported in the case of bugs etc.

But if you'd prefer use Ferrum or similar you are of course welcome to fork the project! 😄
